tutorial/source/bayesian_regression.ipynb

```diff
@@ -80,7 +80,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note that we generate the data with a fixed observation noise $\\sigma = 0.1$.\n",
+    "Note that we generate the data with a fixed observation noise $\\scale = 0.1$.\n",
```

Comment:
This use of `\sigma` is actually greek, so I think we want to keep it. I expect this PR will result in a latex error.

Reply:
Oops..didn't realize the latex incantations here. Will revert these.